dunwoody_disaster/CharacterFactory.py

```diff
@@ -9,7 +9,10 @@ def __init__(self):
         self.level = 0
         self.name = ""
         self.classType = ""
+        self.strength = 0
+        self.intelligence = 0
```

Comment:
What are these?

Reply:
added damage for physical vs magic items